Geometry/HGCalCommonData/src/HGCalWaferMask.cc

```diff
@@ -1277,23 +1277,109 @@ std::vector<std::pair<double, double> > HGCalWaferMask::waferXY(
 
 	Depending on the wafer type and placement index, the corners
 	are chosen in the variable *np*
+        The points 24-35 are the same as points 12-23 with different offset
   */
-  double dx[24] = {HGCalTypes::c00 * delX,  HGCalTypes::c10 * delX,  HGCalTypes::c10 * delX,  HGCalTypes::c00 * delX,
+  const double Sin_60 = std::sqrt(3.0) / 2.0;
```

Comment:
@bsunanda , sorry for being picky, but why not to make them static class members as in 
https://github.com/cms-sw/cmssw/blob/d9244b054c44994b035db8378b7f44c1b5dc9b2b/Geometry/HGCalCommonData/interface/HGCalCellUV.h#L40-L42
so that they don't have to be reassigned every time this method is run?